sdk/webpubsub/Azure.Messaging.WebPubSub/Azure.Messaging.WebPubSub.sln

```diff
@@ -1,14 +1,16 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29709.97
+# Visual Studio Version 17
```

Comment:
WebPubSub files will be removed prior to merging - they are currently here to illustrate changes we need to make to the generator to use these changes to Core.

Reply:
Gone.